src/pages/home/wallet/Portfolio.tsx

```diff
@@ -1,15 +1,29 @@
-import { FunctionComponent, useState } from "react";
-import { SongRoyalties, mockSongRoyalties } from "common";
-import { Box, CircularProgress  } from "@mui/material";
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { FunctionComponent, useEffect, useRef, useState } from "react";
+import { SongRoyalties, mockSongRoyalties, useWindowDimensions } from "common";
+import { Box } from "@mui/material";
+import SkeletonTable from "elements/skeletons/TableSkeleton";
+import theme from "theme";
 import AllCaughtUp from "./AllCaughtUp";
 import SongRoyaltiesList from "./SongRoyaltiesList";
 
 const Portfolio: FunctionComponent = () => {
   const { data = [], isLoading, isSuccess } = mockSongRoyalties;
   const songRoyalties: SongRoyalties[] = data;
-
+  const windowHeight = useWindowDimensions()?.height;
+  const windowWidth = useWindowDimensions()?.width;
+  const maxListWidth = 700;
+  const skeletonRef = useRef<any>();
```

Comment:
It might be better to add the eslint no-explicit-any to just this one line and not the whole file